docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md

```diff
@@ -91,22 +91,26 @@ For any third-party Cron scheduling tools, you can disable Crontab and add the f
 Disabling Crontab creates alerts after the universe is created, but they can be ignored. You need to ensure Cron jobs are set appropriately for YBA to function as expected.
   {{< /tip >}}
 
-- Verify that Python 3.6 or later is installed.
+- Verify that Python 3.5-3.8 is installed. v3.6 is recommended.
 
-    If you are using Python v3.11 or later, install the selinux python package as follows:
+    In case there is more than one Python 3 version installed, ensure that `python3` refers to the right one. For example:
 
     ```sh
-    python3.11 -m pip install selinux
+    sudo alternatives --set python3 /usr/bin/python3.9
+    sudo alternatives --display python3
+    python3 -V
     ```
 
-    In case there is more than one Python 3 version installed, ensure that `python3` refers to the right one as follows:
+    If you are using Python later than v3.6, install the [selinux](https://pypi.org/project/selinux/) package corresponding to your version of python. For example, using [pip](https://pip.pypa.io/en/stable/installation/), you can install as follows:
 
     ```sh
-    sudo alternatives --set python3 /usr/bin/python3.9
-    sudo alternatives --display python3
-    python3 -V
+    python3 -m pip install selinux
     ```
 
+    Refer to [Ansible playbook fails with libselinux-python aren't installed on RHEL8](https://access.redhat.com/solutions/5674911) for more information.
+
+    If you are using Python later than v3.7, set the **Max Python Version (exclusive)** Global Configuration option to the python version. Refer to [Manage runtime configuration settings](../../administer-yugabyte-platform/manage-runtime-config/). Note that only a Super Admin user can modify Global configuration settings.
+
 - Enable core dumps and set ulimits, as follows:
 
```

Comment:
I think we should also link to the other doc here, like you added above. At least temporarily that will communicate these both have similar steps. Maybe say something like "if you are not planning on providing YBA with root access too your node you must also follow the manual provisioning steps here"